tests/modules/misc/fontconfig/simple.nix

```diff
@@ -0,0 +1,28 @@
+{ config, pkgs, ... }:
+
+{
+  config = rec {
```

Comment:
No need for a `config` attribute, the `conf` variable can be moved to a surrounding let expression.
```suggestion
```